content/zh/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md

```diff
@@ -39,7 +39,7 @@ Together, these two ways of exposing Pod and Container fields are called the
 有两种方式可以将 Pod 和 Container 字段呈现给运行中的容器：
 
 * 环境变量
-* [卷文件](/docs/resources-reference/{{< param "version" >}}/#downwardapivolumefile-v1-core)
+* [卷文件](/zh/docs/resources-reference/{{< param "version" >}}/#downwardapivolumefile-v1-core)
```

Comment:
In English, `/docs/resources-reference` is a redirection. We don't have such redirection for Chinese. So this revision will not work. The link is still a 404.

The correct link target is now `/zh/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#the-downward-api`.

Reply:
thank you for suggestion,
suggestion applied.